src/main/kotlin/net/emdal/tank/clause/ReturnClause.kt

```diff
@@ -0,0 +1,12 @@
+package net.emdal.tank.clause
+
+import net.emdal.tank.Entity
+
+class ReturnClause(
+  override var query: List<String> = listOf("RETURN "),
+  override var aliases: Map<String, List<Entity>> = emptyMap()

```

Comment:
Aliases are not in use anywhere. The type inference between the string alias and the entity types do no carry on between clauses which means this is most likely useless to keep adding to the clauses. I think we should remove the `aliases` field from Clause and rather add it back in if needed for something in the future.